### ISSUE TITLE
Issue with breathe 4.12.0 and C++11 member initialization

### ISSUE THREAD

**Issue by rouault** [michaeljones/breathe#425]
I got errors like
```
/Users/travis/build/OSGeo/proj.4/docs/source/development/reference/cpp/io.rst:6:Parsing of expression failed. Using fallback parser. Error was:
Error in postfix expression, expected primary expression or type.
If primary expression:
  Invalid definition: Expected identifier in nested name. [error at 67]
    std::string osgeo::proj::io::AuthorityFactory::CRSInfo::authName = {}
    -------------------------------------------------------------------^
If type:
  Invalid definition: Expected identifier in nested name. [error at 67]
    std::string osgeo::proj::io::AuthorityFactory::CRSInfo::authName = {}
    -------------------------------------------------------------------^
```
with constructs like
```
struct Foo {
     /** my bar */
    std::string bar{};
};
```

The fix I had to do in https://github.com/OSGeo/proj.4/pull/1325/files illustrates hopefully the issue in the practical case I hit

**Comment by jakobandersen** [comment]
This is a problem in Sphinx, which should now be fixed in the next beta version of 2.0.

**Comment by rouault** [comment]
Thanks for the quick fix ! Closing
